webapp/src/package.ts

```diff
@@ -665,9 +674,30 @@ data.mountVirtualApi("open-pkg-meta", {
             r.diagnostics = [];
             files.filter(f => !!f.diagnostics).forEach(f => r.diagnostics.concat(f.diagnostics));
         }
-        const hasGit = f.getPkgId() == "this" && !!(f.header && f.header.githubId)
-        if (hasGit)
-            r.numFilesGitModified = files.filter(f => f.baseGitContent != f.content).length;
+        return r;
+    }
+})
+
+export interface PackagetGitStatus {
```

Comment:
```suggestion
export interface PackageGitStatus {
```

Think I got all the places it's refd